internal/sysfs/readfs.go

```diff
@@ -71,7 +72,7 @@ func (r *ReadFS) Utimens(path string, atim, mtim int64) experimentalsys.Errno {
 }
 
 // compile-time check to ensure readFile implements api.File.
-var _ experimentalsys.File = (*readFile)(nil)
+var _ fsapi.File = (*readFile)(nil)
```

Comment:
this was intentionally implementing only the experimental public VFS API. In fact I think this experimental VFS should not expose OpenAt

Reply:
Thanks for the context! I had thought it was an oversight.